src/lib/components/workflow/dropdown-filter/start-time.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import DropdownMenu from '$lib/components/dropdown-menu.svelte';
+  import { workflowFilters, workflowSorts } from '$lib/stores/filters';
+  import Sort from './sort.svelte';
+
+  $: typeSort = $workflowSorts.find((s) => s.attribute === 'StartTime');
+</script>
+
+<DropdownMenu value={typeSort?.value ?? ''} keepOpen right>
```

Comment:
After the search input is cleared the `Start Time` and `End Time` filters are no longer applied, but still appear selected.

![Screen Shot 2022-10-12 at 10 22 52 AM](https://user-images.githubusercontent.com/15069288/195396800-148c3676-7592-4525-a326-e0c16927c1a8.png)